test/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -82,5 +83,34 @@ public void It_respects_opt_outs(string attributeToOptOut)
 
             actualInfo.Should().Equal(expectedInfo);
         }
+
+        [Theory]
+        [InlineData("netcoreapp1.0")]
+        [InlineData("net45")]
+        public void It_respects_version_prefix(string targetFramework)
+        {
+            if (targetFramework == "net45" && !RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
+            }
+
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("HelloWorld", identifier: targetFramework)
```

Comment:
Does the identifier need to be unique by itself?  Or is something else appended to it to make it unique.

If it has to be unique, just the TFM probably isn't enough.